.rubocop.yml

```diff
@@ -0,0 +1,112 @@
+# Use RuboCop RSpec
```

Comment:
I'm not sure how useful this is - the whole project has A LOT of warnings, and we probably won't ever fix them as it would require major rewrites

Reply:
will create a new PR for rubocop and rspec sytle changes. we can release a new version after 3.0.0